src/constants/GlobalConstants.js

```diff
@@ -6,6 +6,22 @@ import FullyQualifiedName from '../models/FullyQualifiedName';
 
 export const AT_CLASS :string = '@class';
 export const AT_ID :string = '@id';
-
+export const DESTINATION :string = 'dst';
```

Comment:
Instead of adding all of these here, can you add them to a new constants file called `SerializationConstants`? For reference:

https://github.com/openlattice/api/blob/develop/src/main/java/com/openlattice/client/serialization/SerializationConstants.java

Reply:
Oh yes, definitely! I was really unsure where these constants should live.